winpr/libwinpr/crypto/hash.c

```diff
@@ -365,10 +369,32 @@ BOOL winpr_Digest_Init(WINPR_DIGEST_CTX* ctx, WINPR_MD_TYPE md)
 
 	if (mbedtls_md_starts(mdctx) != 0)
 		return FALSE;
-#endif
 
 	return TRUE;
 }
+#endif
+
+BOOL winpr_Digest_Init_MD5_Allow_FIPS(WINPR_DIGEST_CTX* ctx)
```

Comment:
@bjcollins Best make a ```winpr_Digest_InitEx``` function to avoid breaking the current API and let the current ```winpr_DigestInit``` call that one.